Documentation/network/bgp-control-plane.rst

```diff
@@ -73,6 +75,9 @@ Fields
            neighbors[*].peerAddress: The address of the peer neighbor
            neighbors[*].peerASN: The ASN of the peer
 
+       virtualRouters[*].injectedRoutes.prefixes: List of additional routes to be "injected" into announcements.
```

Comment:
Would be nice to have a few more details on injected routes in the doc (thinking of your comment on skipping the LB, or advertising ClusterIPs, from your commit description).